plugins/builds/getBuildStatuses.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const stringSchema = joi.string().regex(/^[0-9]+$/);
+const jobIdsSchema = joi.array().items(stringSchema);
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/builds/statuses',
+    config: {
+        description: 'Get build statuses for jobs',
+        notes: 'Returns id, jobId, and status for builds',
+        tags: ['api', 'builds'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'build', 'pipeline']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const buildFactory = request.server.app.buildFactory;
+            const { jobIds, numBuilds, offset } = request.query;
+
+            const payload = {
+                jobIds: jobIds.map(jobId => parseInt(jobId, 10)),
```

Comment:
these parameters should of been passed in as integer

Reply:
parameters are from request.query they all come in as strings